kolibri/core/apps.py

```diff
@@ -10,8 +10,11 @@
 from kolibri.core.sqlite.pragmas import CONNECTION_PRAGMAS
 from kolibri.core.sqlite.pragmas import START_PRAGMAS
 from kolibri.core.sqlite.utils import repair_sqlite_db
+from kolibri.core.utils.cache import RedisSettingsHelper
+from kolibri.utils.conf import OPTIONS
 
 logger = logging.getLogger(__name__)
+CACHE_OPTIONS = OPTIONS["Cache"]
```

Comment:
It's probably OK, but for safety, I'd recommend doing this inside the `check_redis_settings` method.

For context, `OPTIONS` is a lazily evaluated object that relies on the Kolibri plugin registry having been initialized, this should have happened before Django starts importing apps into its app registry, but something may go oddly with import order that might cause this module to be imported earlier.

Reply:
Good call, I'll make that change.